packages/avatars/src/styled/StyledAvatar.spec.tsx

```diff
@@ -26,58 +27,60 @@ describe('StyledAvatar', () => {
   });
 
   it('renders system styling if provided', () => {
-    const { container } = render(<StyledAvatar isSystem />);
+    const { container } = render(<StyledAvatar $isSystem />);
 
     expect(container.firstChild).toHaveStyleRule('border-radius', DEFAULT_THEME.borderRadii.md);
   });
 
   describe('color', () => {
     it('renders surface color as expected', () => {
-      const { container } = render(<StyledAvatar status="away" surfaceColor="red" />);
+      const { container } = render(<StyledAvatar $status="away" $surfaceColor="red" />);
 
-      expect(container.firstChild).toHaveStyleRule('color', 'red');
+      expect(container.firstChild).toHaveStyleRule('color', 'red', {
+        modifier: css`&&`
```

Comment:
Interesting .. does `'&&'` not work here?

Reply:
Ah, it does. I had assumed it wouldn't given the transform provided by the `css` function.